bridge/client/app/_components/ktb-evaluation-details/ktb-evaluation-details.component.html

```diff
@@ -1,7 +1,11 @@
 <div *ngIf="evaluationData">
   <div class="mt-1 mb-1" *ngIf="showChart && !isInvalidated">
     <div fxLayout="row" class="pr-2">
-      <div fxFlex></div>
+      <div fxFlex>
+        <button dt-button variant="secondary" color="warning" *ngIf="updateResults" (click)="refreshEvaluationBoard(updateResults)">
```

Comment:
We already have a refresh button on several screens (dashboard, sequence view, ...). I would suggest reusing that button to keep consistent. Also, I think the warning color is not needed here.
```
<button dt-button (click)="refreshEvaluationBoard(updateResults)" *ngIf="updateResults">
  <dt-icon name="refresh"></dt-icon>
  Reload
</button>
```